metricbeat/module/jolokia/_meta/fields.yml

```diff
@@ -0,0 +1,10 @@
+- key: jolokia
+  title: "Jolokia"
+  description: >
+    Jolokia Module
+  fields:
```

Comment:
Can you add an experimental flag here? https://github.com/elastic/beats/blob/master/metricbeat/module/kafka/_meta/fields.yml#L6